### PR TITLE
Xcode 11.2 Support.

### PR DESCRIPTION
**Pull request by sebastianv1**
* Exposes public `factory` function for assisted injection `Factory` class. This is similar to `Provider` and other core primitives to allow for manual testing.
* Tests for the sample project weren't being run (or built). Change travis CI config to build and test the sample project.
* Bump TravisCI to Xcode 11.2.

### CODE REVIEW

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (DJBen) [inline .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata]: Does this need to be in the commit?

File: Examples/CleanseGithubBrowser/CleanseGithubBrowserTests/CleanseGithubBrowserTests.swift [hunk 2]
```diff
@@ -14,10 +14,13 @@ import Cleanse
 class CleanseGithubBrowserTests: XCTestCase {
     func testMembersViewController_Title() {
         // Tests that the member's view controller sets the title appropriately.
+        let factory = Factory<AssistedInjectionTags.MembersPageSettingsSeed> { _ in
```
Comment (DJBen) [inline Examples/CleanseGithubBrowser/CleanseGithubBrowserTests/CleanseGithubBrowserTests.swift]: Interesting. I am wondering how do you deal with the `seed`

Comment (sebastianv1) [reply Examples/CleanseGithubBrowser/CleanseGithubBrowserTests/CleanseGithubBrowserTests.swift]: Yep it is! For the purposes of this test case I don't care about the seed.

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata (hunk 1 above)
Comment (sebastianv1) [reply .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata]: Nope. Removing

**Pull request merged**